src/Button/GeoLocationButton/GeoLocationButton.jsx

```diff
@@ -99,7 +138,7 @@ class GeoLocationButton extends React.Component {
     onGeolocationChange: () => undefined,
     onError: () => undefined,
     showMarker: true,
-    follow: true,
+    follow: false,
```

Comment:
In my opinion `follow: true` is a good default, but you decide.

Reply:
Following is somewhat buggy atm. I think this needs a further refactoring. I'll stick with fals in the meanwhile.

Reply:
Could you please create an issue regarding to this?

Reply:
https://github.com/terrestris/react-geo/issues/755